src/main/java/org/broadinstitute/consent/http/resources/NihAccountResource.java

```diff
@@ -0,0 +1,52 @@
+package org.broadinstitute.consent.http.resources;
+
+import org.broadinstitute.consent.http.models.NIHUserAccount;
+import org.broadinstitute.consent.http.service.NihAuthApi;
+
+import javax.annotation.security.RolesAllowed;
+
+import javax.ws.rs.Path;
+import javax.ws.rs.POST;
+import javax.ws.rs.Produces;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.core.Response;
+
+@Path("api/nih-register/")
```

Comment:
We generally camel case paths . We should try to be consistent here.

Reply:
Got it, changed  Ui and backend paths as Greg recommended.